build-system/compile/bundles.config.js

```diff
@@ -258,7 +258,7 @@ exports.extensionBundles = [
   },
   {
     name: 'amp-accordion',
-    version: '0.1',
+    version: ['0.1', '1.0'],
```

Comment:
Note: If you don't want to include the empty css file, you can create a separate entry for `1.0` like amp-fit-text:
https://github.com/ampproject/amphtml/blob/88a66ceed065edd023e1a78acc03d593a07634be/build-system/compile/bundles.config.js#L546-L558

Reply:
Sure. But I figured there will be some CSS soon enough. I just didn't get to porting that part yet.